README.md

```diff
@@ -240,6 +251,11 @@ sharing Decomposition 4, 2 sharing Decomposition 5, and 4 sharing Decomposition
        [-g num] Number of IO groups (subfiles) (default 1)
        [-o output_dir] Output directory name (default ./)
        [-i target_dir] Path to directory containing the input files
+              If set, E3SM I/O will read the data from the input files and write it out in the write test.
+              The input files must be generated by E3SM under the same configuration (decomposition file) as the test case and be written by the same underlying I/O library.
```

Comment:
`The input files must be named the same as the output file`
Does this mean the input file will be overwritten by the output files?
Why not just different input and output file names?

Reply:
They are in the different folder so it won't be overwritten.
The output file name cannot be set by the user, the user can only set the folder that contains them.
We assumes the same for input files.

Reply:
Let's change that to let users to specify input and output file names.
For ADIOS blob method the output file name is a folder.
We can also set default values.

Reply:
It should be a different commit as this is about updating the descriptions.
What options do we want to use? There can be more than 1 file.

Reply:
In that case, create a new PR for that and I will merge that before this.

-i path:  path to input file or folder
Similar for -o.
We will describe their different usages when different API/strategy is used.

Reply:
F case have 2 files. How will user assign them?

Reply:
Append 'h0' and 'h1' but before '.nc'.